src/easyscience/Objects/virtual.py

```diff
@@ -16,6 +16,12 @@
 
 from easyscience import global_object
 from easyscience.Constraints import ObjConstraint
+from easyscience.Objects.variable.descriptor_base import DescriptorBase
```

Comment:
not reviewing this file, since it is going to be deleted, so as long as the tests pass, I assume it is fine.